filebeat/processor/add_kubernetes_metadata/matchers.go

```diff
@@ -38,6 +40,7 @@ func init() {
 }
 
 const LogPathMatcherName = "logs_path"
+const pathSeparator = string(os.PathSeparator)
```

Comment:
os.PathSeparator is already a const.
Probably there is no point redefining here, unless you think it makes things more clear. I'd assume people reading this will be glad to find old good `os.PathSeparator` vs our own const

Reply:
I guess the original author wanted to avoid casting to `string` on every single use?

Reply:
I didn't thought of that.
ok